bitnami/mariadb/values.yaml

```diff
@@ -185,6 +185,9 @@ primary:
     log-error=/opt/bitnami/mariadb/logs/mysqld.log
     character-set-server=UTF8
     collation-server=utf8_general_ci
+    slow_query_log=1
```

Comment:
I would set this `=0` by default. So it is not enabled always and it would be easy for an user to enable it if needed.